tensorflow/workspace.bzl

```diff
@@ -508,6 +508,14 @@ def tf_workspace(path_prefix="", tf_repo_name=""):
   )
 
   native.new_http_archive(
+    name = "lmdb",
```

Comment:
@bowang You need to add a line "@lmdb//:LICENSE" here: https://github.com/tensorflow/tensorflow/blob/master/tensorflow/tools/pip_package/BUILD

(I'll make sure it gets added in the appropriate place internally to make it stick.)